src/components/SignUpPopup.js

```diff
@@ -0,0 +1,35 @@
+import {BaseComponent} from "../pageObjects/BaseComponent.js";
+import {expect} from "@playwright/test";
+
+export default class SignUpPopup extends BaseComponent {
+        
+    constructor(page) {
+        super(page,page.locator('div.modal-dialog'));
+        this.nameInput = this._container.locator('input#signupName')
+        this.lastNameInput = this._container.locator('input#signupLastName')
+        this.emailInput = this._container.locator('input#signupEmail')
+        this.passwordInput = this._container.locator('input#signupPassword')
+        this.repasswordInput = this._container.locator('input#signupRepeatPassword')
+        this.registerButton = this._container.locator('.btn-primary')
+        this.nameErrorMessage = this._container.locator('.invalid-feedback')
```

Comment:
this.nameErrorMessage - it can be an error related to any of fields based on your selector, rename this property